models/fivetran_log.yml

```diff
@@ -33,14 +33,18 @@ models:
       - name: connector_health
         description: Status of the connector's data flow. Can be `"broken"`, `"incomplete"`, `"connected"`, `"paused"`, `"initial sync in progress"`, or `"priority first sync"`.
 
+      - name: last_successful_sync_completed_at
+        description: > 
+          Timestamp of when the last successful sync completed.
+
       - name: last_sync_started_at
         description: > 
           Timestamp of when the last sync was started. Note that if new data comes in after a sync has 
           begun, the new data will not be included in this sync.
 
       - name: last_sync_completed_at
         description: > 
-          Timestamp of when the last sync was complted.
+          Timestamp of when the last sync was completed. This could be either a failure or a successful sync.
```

Comment:
what about `Timestamp of when the last sync attempt was completed...`

Reply:
Yeah I think I over explained this one haha I think before we alluded to the fact that the timestamp being completed meant it succeeded, but that is not the case anymore. I like the idea of keeping it simple.

Reply:
oh no i think we should keep the second sentence! the `...` included that haha

Reply:
so like `Timestamp of when the last sync attempt was completed. This could be either a failure or a successful sync.`

Reply:
Ahhh the word attempt is what I was missing here. Yeah I like the original haha thanks!

Reply:
oh and a reminder to regenerate the docs afterward!